pages/index.vue

```diff
@@ -1,19 +1,82 @@
 <template>
-  <vue-layer-map :initial-zoom="6" :initial-center="[4.356998572,50.855996576]">
+  <vue-layer-map :initial-zoom="6" :initial-center="[longitude, latitude]">
     <template #features>
-      <vue-layer-marker :coordinates="[4.356998572,50.855996576]" />
+      <vue-layer-marker :coordinates="[longitude, latitude]" />
     </template>
   </vue-layer-map>
 </template>
 
 <script lang="ts">
-import Vue from "vue";
-import VueLayerMarker from "~/components/VueLayerMarker.vue";
+    import Vue from "vue";
+    import { connect } from "mqtt";
+    import VueLayerMarker from "~/components/VueLayerMarker.vue";
 
-export default Vue.extend({
-  name: "IndexPage",
-  components: { VueLayerMarker }
-});
+    export default Vue.extend({
+        name: "IndexPage",
+        components: { VueLayerMarker },
+        data () {
+            return {
+                longitude: 0,
+                latitude: 0
+            };
+        },
+        mounted () {
+            this.connectToClient();
+        },
+        methods: {
+            connectToClient () {
+                const token = process.env.FLESPI_KEY || "not found"; // TODO : Handle when the key isn't there
+                if (token.length !== 64) {
```

Comment:
This is something that can be done in a `try/catch` block. As it is an misconfiguration, this could be handled with `console.error()`, but any solution without `console.log()` is good.